### PR TITLE
chore(Videoland): rewrite presence & fix null & undefined issues

### PR DESCRIPTION
**Pull request by DarkVillager**
## Description 

- Fix issues
- Rewrite presence
## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

File: websites/V/Videoland/presence.ts [hunk 1]
```diff
@@ -3,70 +3,100 @@ const presence = new Presence({
...
 			presence.getSetting<boolean>("covers"),
-		]);
+		]),
+		video = document.querySelector("video"),
+		getInfo = document.querySelectorAll('[type="application/ld+json"]')?.[1]
+			?.innerHTML
+			? JSON.parse(
+					document.querySelectorAll('[type="application/ld+json"]')?.[1]
+						?.innerHTML
+			  )
```
Comment (theusaf) [inline websites/V/Videoland/presence.ts:27]: Could `textContent` be used instead?

**Review by theusaf — CHANGES_REQUESTED** [review]

File: websites/V/Videoland/presence.ts (hunk 1 above)
Comment (DarkVillager) [reply websites/V/Videoland/presence.ts:27]: No because its the innerhtml

Comment (DarkVillager) [reply websites/V/Videoland/presence.ts]: Nvm it can be used.

**Review by theusaf — APPROVED** [review]

File: websites/V/Videoland/presence.ts [hunk 2]
```diff
@@ -3,70 +3,100 @@ const presence = new Presence({
...
+		presenceData.details = `Bekijkt ${document
+			.querySelector('[class*="is-active"]')
+			?.textContent?.toLowerCase()}`;
+	} else if (document.querySelector("article")) {
+		presenceData.details = `Bekijkt: ${
+			document
+				.querySelector<HTMLMetaElement>("meta[property='og:title']")
+				?.content.toLowerCase()
+				.split("on videoland")[0]
+				.split("op videoland")[0]
```
Comment (Bas950) [inline websites/V/Videoland/presence.ts:91]: Split on the same thing twice?

Comment (DarkVillager) [reply websites/V/Videoland/presence.ts:91]: > Split on the same thing twice?
on / op

**Pull request merged**